scripts/lean-pr-testing-comments.sh

```diff
@@ -0,0 +1,105 @@
+## Create comments and labels on a Lean 4 PR after CI has finished on a `lean-pr-testing-NNNN` branch.
```

Comment:
```suggestion
## Create comments and labels on a Lean 4 PR after CI has finished on a `lean-pr-testing-NNNN` branch.
set -e
```
unless the script doesn't work with this

Reply:
I'd prefer not to make behaviour-changing changes without good reason, given the lengthy cycle to test this script.

Reply:
This one is reasonably likely to break something, so I'm ok with skipping it.

Reply:
Since I've just fired up a test run, I'll include this.